Makefile

```diff
@@ -67,7 +67,8 @@ build: gen docs ## Download dependencies and build the default binary
 	$(GOBUILD) -o $(LAKECTL_BINARY_NAME) -ldflags $(LD_FLAGS) -v ./cmd/$(LAKECTL_BINARY_NAME)
 
 lint: ## Lint code
-	$(DOCKER) run --rm -v $(CURDIR):/app -w /app golangci/golangci-lint:$(GOLANGCILINT_VERSION) golangci-lint run -v
+	go get github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCILINT_VERSION)
```

Comment:
This changes `go.mod` and `go.sum`.  So if I ever run it I still change a foreign file.

Let's do it a bit differently!
1. Add `golangci-lint` to `go.mod` (i.e. run `go get ...` once) and _commit the file_.
2. Run the lint step here.

Now we have:
- A repeatable linting step.  E.g. suppose the linter adds some random stupid step.  We don't want it.  So we're already pinning the version -- let's use go.mod to do that!
- Clean linting.  No changes to files due to linting.
- (Minor) Faster linting.

WDYT?

Reply:
adding tools.go and use go get for out code gen and lint tools - it will track the version and should run on all platforms.